app/views/posts/_expanded.html.erb

```diff
@@ -295,27 +295,11 @@
 
         <% if is_question && post.children.length >= SiteSetting["TableOfContentsThreshold"] && SiteSetting["TableOfContentsThreshold"] != -1 %>
           <div class="toc has-margin-left-4" id="toc-toggle">
-            <button class="toc--header" data-toggle="#toc-toggle" data-toggle-property="class" data-toggle-value="is-active">Table of Contents</button>
-            <% post.children.each do |answer| %>
-              <a href="<%= generic_share_link(answer) %>" class="toc--entry">
-                <div class="toc--badge">
-                  <span class="badge is-tag is-green">+<%= answer.upvote_count %></span>
-                </div>
-                <div class="toc--badge">
-                  <span class="badge is-tag is-red">&minus;<%= answer.downvote_count %></span>
-                </div>
-                <div class="toc--full">
-                  <% sanitised = strip_markdown(answer.body_markdown) %>
-                  <% max_length = SiteSetting["TableOfContentsExcerptLength"] %>
-                  <p>
-                    <%= sanitised[ 0 ... max_length ] %>
-                    <% if sanitised.length > max_length %>
-                      &hellip;
-                    <% end %>
-                  </p>
-                  <p class="has-font-size-small"><%= time_ago_in_words(post.updated_at, locale: :en_abbrev) %> ago</p>
-                </div>
-              </a>
+            <button class="toc--header" data-toggle="#toc-toggle" data-toggle-property="class" data-toggle-value="is-active">Show More</button>
+            <% sorted_answers = post.children.sort_by { |answer| answer.score }.reverse! %>
+            <% sorted_answers.each do |answer| %>
+              <% next if answer.deleted? && !moderator? %>
```

Comment:
nitpick (non-blocking): this _only_ allows moderators to see deleted posts, not all users with the privileges to view deleted. Maybe that's what we want; don't know.

Reply:
For consistency, it'd be better if people who can see deleted posts farther down the page could also see them here.  Otherwise we're going to confuse people and probably get bug reports.  We'd want them to go to the bottom, like on the main page, though -- is that hard?